roles/runtime/defaults/main.yml

```diff
@@ -25,6 +25,12 @@ run__namespace_cdp:                 "{{ common__namespace_cdp }}"
 run__env_name:                      "{{ common__env_name }}"
 run__datalake_name:                 "{{ common__datalake_name }}"
 run__vpc_name:                      "{{ common__vpc_name }}"
+run__vpc_public_subnets_suffix:     "{{ common__vpc_public_subnets_suffix }}"
+run__vpc_private_subnets_suffix:    "{{ common__vpc_private_subnets_suffix }}"
+run__vpc_public_subnets_name:       "{{ [run__namespace, run__vpc_public_subnets_suffix] | join('-') }}*"
```

Comment:
Pattern should be a configuration variable, like `infra.vpc.public_subnets.name`, and then a default if that variable is not present in the inbound config.  Otherwise, variables like these two should go in `vars/main.yml`, as they are not "user serviceable parts."